vllm/config/speculative.py

```diff
@@ -215,7 +215,8 @@ def hf_config_override(hf_config: PretrainedConfig) -> PretrainedConfig:
 
         return hf_config
 
-    def __post_init__(self):
+    @model_validator(mode="after")
+    def _configure(self) -> Self:
```

Comment:
Most of this is post initialisation, not validation. 

```suggestion
    def __post_init__(self):
```